Libraries/Image/ImageBackground.js

```diff
@@ -57,9 +57,16 @@ class ImageBackground extends React.Component<$FlowFixMeProps> {
   render(): React.Node {
     const {children, style, imageStyle, imageRef, ...props} = this.props;
 
+    // If `no` is specified for `importantForAccessibility`, it will be changed to `no-hide-descendants` because the text inside should not be focused.
```

Comment:
@kacieb , do you think this same logic makes sense for "ImageBackground"? I am not sure where/how this component is commonly used, but if people tend to consider it a wrapper element that does not impact it's children, I don't think this makes sense necessarily.

Basically, this would cause all children passed into this background to not be focusable even if you *only* wanted the wrapper itself to not be focusable. Since you are passing these children in yourself (unlike button or picker), this doesn't seem to make as much sense to me.

Reply:
Based on this RNTester example I found, one possible use case of `ImageBackground` is to nest content:
![176247006_194646549153965_8706847766633327816_n](https://user-images.githubusercontent.com/25069965/115599512-f3909680-a298-11eb-956c-021912fb3af3.jpg)

So it seems to me like we want to allow children to be able to be focusable, still, or at least allow the React Native user to choose which they'd prefer.

cc @lunaleaps in case you have any additional context on Image that I'm missing.

Reply:
Sorry had missed this comment. Agreed, we should offer the flexibility for focussed children elements nested in an ImageBackground. All my context comes from is the docs: https://reactnative.dev/docs/next/imagebackground